stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt

```diff
@@ -348,9 +351,10 @@ internal class BillingAddressView @JvmOverloads constructor(
             // country will trigger a validation of the postal code, which will be
             // invalid if not set first.
             this.postalCodeView.setText(it.postalCode)
-            it.country?.let { countryCode ->
-                countryLayout.selectedCountry = CountryUtils.getCountryByCode(countryCode)
-                this.countryView.setText(CountryUtils.getDisplayCountry(countryCode))
+
+            it.country.takeIf { !it.isNullOrBlank() }?.let { countryCode ->
```

Comment:
Looking at this line, it was hard to tell what a `country` vs `countryCode` vs `CountryCode(countryCode)` represented.

It might be nice to add an `internal countryCode: CountryCode? get() = country?.let { CountryCode(it) }` to `Address` to make this more concise.